java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1658,6 +1658,30 @@ public void zrank(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void zrevrank(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.5, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key, membersScores).get());
+        assertEquals(0, client.zrevrank(key, "three").get());
+
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0")) {
```

Comment:
Can we add a comment explaining that the WITHSCORE arg was added in 7.2.0?

Reply:
I dont think we mention them in other commands. Didn't see AWS team adding any references to Redis Versions.